R/calcOutput.R

```diff
@@ -430,8 +434,13 @@ calcOutput <- function(type, aggregate = TRUE, file = NULL, years = NULL, round
   if (!is.null(round)) {
     if (x$class != "magpie") stop("rounding can only be used in combination with x$class=\"magpie\"!")
     x$x <- round(x$x, round)
+  } else if (!is.null(signif)) {
```

Comment:
```suggestion
  }
  if (!is.null(signif)) {
```

Reply:
I dislike committing from the web interface.  Will force-push from remote.